.travis.yml

```diff
@@ -25,7 +25,7 @@ before_script:
   - export DISPLAY=:99.0; sh -e /etc/init.d/xvfb start ;
   - google-chrome-stable --headless --disable-gpu --remote-debugging-port=9222 http://localhost \&  ;
 before_install:
-  - curl -o- -L https://yarnpkg.com/install.sh | bash
+  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.0.2
```

Comment:
Can we do the same for Appveyor too? Let me check...

Reply:
It is not yet available under Windows. See: [here](https://chocolatey.org/packages/yarn#comment-3534223070).

Reply:
sure, if you know how

Reply:
or let's wait till it hurts?

Reply:
Let's wait.